projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -140,7 +166,7 @@ public function get_items_permissions_check( $request ) { // phpcs:ignore Variab
 	 * @return WP_REST_Response
 	 */
 	public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-		return rest_ensure_response( wp_get_scheduled_events( 'jetpack_scheduled_update' ) );
+		return rest_ensure_response( Scheduled_Updates::get_scheduled_events_with_statuses() );
```

Comment:
Should we follow Core convention here and pass the results of `wp_get_scheduled_events( 'jetpack_scheduled_update' )` through a [prepare_item_for_response](https://github.com/WordPress/WordPress/blob/master/wp-includes/rest-api/endpoints/class-wp-rest-plugins-controller.php#L578) method that calls `add_additional_fields_to_object` and lets us add the status information in the same way as we add `is_managed` to the plugin endpoint?